### PR TITLE
Add Readme

### LINKED ISSUE #7: add a readme
Add a readme

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

Update readme.

Fixes #7

### CODE REVIEW

**Comment by yeya24** [comment]
PTAL @zhouqiang-cl

File: README.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+## Getting Started
+
+### Use Grafana image with Chaos Mesh data source
+
+Since Chaos Mesh data source is not officially released, it is not supported to install it from [Grafana Plugins Hub](https://grafana.com/grafana/plugins).
+
+But you can still try to use the customized Grafana Docker image, with the Chaos Mesh data source built-in.
+
+```bash
+docker pull yeya24/grafana:chaos
```
Comment (zhouqiang-cl) [inline README.md:16]: We can later push to pingcap/grafana:chaos, since chaosmesh is already token..

**Review by zhouqiang-cl — APPROVED** [review]
rest LGTM

File: README.md (hunk 1 above)
Comment (yeya24) [reply README.md:16]: This image is only for testing. After this data source is officially ready, we can directly install it at Grafana. So no additional images are required.

**Pull request merged**